packages/block-library/src/navigation/edit/index.js

```diff
@@ -247,6 +247,7 @@ function Navigation( {
 		status: classicMenuConversionStatus,
 		error: classicMenuConversionError,
 		value: classicMenuConversionResult,
+		classicInnerBlocks: classicMenuConversionInnerBlocks,
```

Comment:
This is used to pass the innerBlocks to the block from the hook.

Reply:
This changed a bit with #43081 and will change more in #42956.

But I think its ok.  `convert` can return the `wp_navigation` and the `innerBlocks`:
```js
const { navigationMenu, innerBlocks } = await convertClassicMenu( classicMenu );
setRef( navigationMenu.id );
replaceInnerBlocks( innerBlocks );
```